test/manager_test.exs

```diff
@@ -31,7 +31,7 @@ defmodule ManagerTest do
 
   test "it is possible to add more workers to a spider" do
     spider_name = Manager.TestSpider
-    :ok = Crawly.Engine.start_spider(spider_name)
+    :ok = Crawly.Engine.start_spider(spider_name, crawl_id: "add_workers_test")
```

Comment:
the engine should generate a crawl_id automatically. the change is unnecessary and isn't part of the test scope